docs/sql/index.md

```diff
@@ -1,407 +1,391 @@
-
-> **DROP TABLE** _table_name_;
+#### FROM Clause
 
-![drop-table](/images/syntax/drop_table.svg)
+```ebnf
+FROM table [ [AS] alias ] { [INNER] JOIN table [ [AS] alias ] ON predicate }
+```
 
 Examples:
-
 ```sql
-DROP TABLE inventory;
+-- Which orders include currently stocked items?
+SELECT order.*
+FROM Orders order
+JOIN Inventory inv
+ON order.item_id = inv.item_id
 ```
 
-## Queries
-
-### `FROM`
-
-A `FROM` clause derives a data source from a table name.
-
-The syntax of the `FROM` clause is:
-
-> **FROM** _table_name_ _join_clause_?;
-
-![from](/images/syntax/from.svg)
+#### WHERE Clause
 
-#### Examples
+See [Subscriptions](#where).
 
-Select all rows from the `inventory` table:
+### INSERT
 
-```sql
-SELECT * FROM inventory;
+```ebnf
+INSERT INTO table [ '(' column { ',' column } ')' ] VALUES '(' literal { ',' literal } ')'
 ```
 
-### `JOIN`
+Examples:
+```sql
+-- Inserting one row
+INSERT INTO Inventory (item_id, item_name) VALUES (1, 'health1');
 
-A `JOIN` clause combines two data sources into a new data source.
+-- Inserting two rows
+INSERT INTO Inventory (item_id, item_name) VALUES (1, 'health1'), (2, 'health2');
+```
 
-Currently, SpacetimeDB SQL supports only inner joins, which return rows from two data sources where the values of two columns match.
+### DELETE
 
-The syntax of the `JOIN` clause is:
+```ebnf
+DELETE FROM table [ WHERE predicate ]
+```
 
-> **JOIN** _table_name_ **ON** _expr_ = _expr_;
+Deletes all rows from a table.
+If `WHERE` is specified, only the matching rows are deleted.
 
-![join](/images/syntax/join.svg)
+Examples:
+```sql
+-- Delete all rows
+DELETE FROM Inventory;
 
-### Examples
+-- Delete all rows with a specific item_id
+DELETE FROM Inventory WHERE item_id = 1;
+```
 
-Select all players rows who have a corresponding location:
+### UPDATE
 
-```sql
-SELECT player.* FROM player
- JOIN location
- ON location.entity_id = player.entity_id;
+```ebnf
+UPDATE table SET [ '(' assignment { ',' assignment } ')' ] [ WHERE predicate ]
 ```
 
-Select all inventories which have a corresponding player, and where that player has a corresponding location:
+Updates column values of existing rows in a table.
+The columns are identified by the `assignment` defined as `column '=' literal`.
+The column values are updated for all rows that match the `WHERE` condition.
+The rows are updated after the `WHERE` condition is evaluated for all rows.
 
+Examples:
 ```sql
-SELECT inventory.* FROM inventory
- JOIN player
- ON inventory.inventory_id = player.inventory_id
- JOIN location
- ON player.entity_id = location.entity_id;
+-- Update the item_name for all rows with a specific item_id
+UPDATE Inventory SET item_name = 'new name' WHERE item_id = 1;
 ```
 
-### `SELECT`
+### SET
 
-A `SELECT` statement returns values of particular columns from a data source, optionally filtering the data source to include only rows which satisfy a `WHERE` predicate.
+> WARNING: This statement is not part of the stable API.
```

Comment:
Can we say "Support for the `SET` statement is experimental" instead? The appeal to "the stable API" reads weird to me.